transcendence-app/src/chat/chatroom/infrastructure/db/chatroom.entity.ts

```diff
@@ -26,15 +27,17 @@ export interface ChatroomData {
 export class Chatroom {
   id: string;
   name: string;
+  @ApiHideProperty()
   @Exclude()
   password: string | null;
   avatarId: string | null;
   avatarX: number = 0;
   avatarY: number = 0;
   createdAt: Date;
   ownerId: string;
+  @ApiProperty({ type: Boolean })
   @Expose()
-  get public(): boolean {
+  get isPublic(): boolean {
```

Comment:
public is a keyword, and becomes _public when openapi generator makes the frontend code. isPublic doesn't, hence, to stay consistent, the change.

Reply:
Good catch!